src/reducers/index.js

```diff
@@ -9,12 +9,14 @@ const initialState = {
 // of our redux store, along with an action created
 // by our action creator. What does the reducer
 // need to do with the count in each case?
-export default (state = initialState, action) => {
+export default (state = initialState.count, action) => {
   switch (action.type) {
     case INCREMENT:
     // Fill in the body of this case
+    return ++state;
```

Comment:
```suggestion
    return {...state, count: state.count +1}
```
The way you're doing it here is not immutable.